components/NewPersonView/UnfinishedSubmissions.tsx

```diff
@@ -43,7 +43,7 @@ const UnfinishedSubmissionsEvent = ({
     data: unfinishedSubmissionRes,
     isValidating,
     error,
-  } = useUnfinishedSubmissions(personId);
+  } = useUnfinishedSubmissions(personId, 1, 4);
```

Comment:
I could see in the future allowing the user to load more, but this maintains the current functionality of just displaying the first 4 unfinished submissions